packages/graphql-yoga/src/plugins/useLandingPage.ts

```diff
@@ -0,0 +1,24 @@
+import { PromiseOrValue } from '@envelop/core'
+import { YogaLogger } from '../logger.js'
+import { Plugin } from './types.js'
+import landingPageBody from '../landing-page.js'
+
+export function useLandingPage<TServerContext>(): Plugin<{}, TServerContext> {
+  return {
```

Comment:
I think we will add the landing/404 page in this plugin;
https://github.com/dotansimha/graphql-yoga/pull/1455/files#diff-82c6efc99edaf736ddb4b80fae5d2cf28760aa07b0fa1a6194e6e692e818b121R10
No?

Reply:
Ok ok :D no worries just wanted to make sure if we are on the same page